components/image/slider/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+lib
```

Comment:
Have in mind that if you do this no 'lib' folder in any subfolder will be tracked. Not sure if we want that, as lib is a common subfolder name. '/lib' would be more accurate.